src/api.jl

```diff
@@ -62,8 +106,12 @@ module API
         )
 
         exp = join(["loadFile", "(", "fileName", "=", modelicaString(fileName), ",", "encoding", "=", modelicaString(encoding), ",", "uses", "=", uses,",", "notify", "=", notify,",", "requireExactVersion", "=", requireExactVersion,")"])
-        return sendExpression(omc, exp)
+        success = OMJulia.sendExpression(omc, exp)
```

Comment:
@AnHeuermann I see that you are using a better error reporting, but in our mos scripts we always use getErrorString()
```
loadFile("XXX"); 
getErrorString();
```
but from users point of view the API should give error msg without the use of getErrorString()